js/src/lib/Details/JSONLDItem.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
This file should be commented as to:
- what is the purpose of it
- does it has any effect

Without knowing what is ldjson it can be a WTF moment to see this.